nix-darwin/default.nix

```diff
@@ -63,8 +63,10 @@ in
       extraSpecialArgs = mkOption {
         type = types.attrs;
         default = { };
+        example = literalExpression "{ inherit emacs-overlay; }";
```

Comment:
Maybe `inherit self` would be a better example, considering that it is already the default first parameter of the Flake :thinking: ?

Reply:
@thiagokokada it is more correct (since the attribute will be always present), but I don't know if it helps the reader much.

I think the fact `emacs-overlay` is obviously not part of the default config kinda helps catch attention.

No super strong feelings though.

Reply:
Maybe put both them.

Reply:
My only problem with putting `emacs-overlay` is that it doesn't make sense for anyone that is not using `emacs-overlay` already and also destructuring it from `inputs`.

`self` is guarantee to exist. We could also use `inputs` (but this one is not).

Reply:
The idea is that it would make sense for everyone, independently of whether they’ve defined it.

It doesn’t have to actually be in scope, since it’s just an example.

Reply:
I think emacs-overlay is as good an example as any. No need to change IMHO.